src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -845,4 +846,57 @@ public void shouldNotExtendLoanDueDateIfOverdueLoanIsRecalled() {
       hasJsonPath("loan.itemStatus", "Checked out"))
     ));
   }
+
+  @Test
+  public void loanDueDateTruncatedOnCheckoutWhenRecallAnywhereInQueue() {
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource requestServicePoint = servicePointsFixture.cd1();
+    final IndividualResource jessica = usersFixture.jessica();
+    final IndividualResource charlotte = usersFixture.charlotte();
+    final IndividualResource james = usersFixture.james();
+    final IndividualResource rebecca = usersFixture.rebecca();
+    final IndividualResource steve = usersFixture.steve();
+
+    final Period loanPeriod = Period.weeks(4);
+    setFallbackPolicies(new LoanPolicyBuilder()
+      .withName("Can Circulate Rolling With Recalls")
+      .withDescription("Can circulate item With Recalls")
+      .rolling(loanPeriod)
+      .unlimitedRenewals()
+      .renewFromSystemDate()
+      .withRecallsMinimumGuaranteedLoanPeriod(Period.weeks(2))
+      .withRecallsRecallReturnInterval(Period.weeks(2)));
+
+      checkOutFixture.checkOutByBarcode(
+        smallAngryPlanet, steve, now(UTC));
+
+      requestsFixture.placeHoldShelfRequest(
+        smallAngryPlanet, james, now(UTC),
+        requestServicePoint.getId(), "Hold");
+      
+      requestsFixture.placeHoldShelfRequest(
+        smallAngryPlanet, jessica, now(UTC),
+        requestServicePoint.getId(), "Hold");
+
+      requestsFixture.place(new RequestBuilder()
+        .recall()
+        .forItem(smallAngryPlanet)
+        .fulfilToHoldShelf()
+        .by(rebecca)
+        .fulfilToHoldShelf()
+        .withPickupServicePointId(requestServicePoint.getId()));
+
+      checkInFixture.checkInByBarcode(smallAngryPlanet);
+
+      final DateTime checkOutDate = now(UTC);
+      final DateTime truncatedLoanDate = checkOutDate.plusWeeks(2);
+
+      final IndividualResource loan = checkOutFixture.checkOutByBarcode(
+        smallAngryPlanet, james, checkOutDate);
+
+      String loanDueDate = loan.getJson().getString("dueDate");
+
+      assertThat(loanDueDate, is(truncatedLoanDate.toString()));
+      
```

Comment:
Please remove this blank line as I don't think it improves the readability of the code